### DISCUSSION TITLE
Let's use slack/gitter for fast communication inside and outside community.

### DISCUSSION THREAD

**Discussion by LaserPhaser** [pytest-dev/pytest#8067] [Ideas] [answered]
Let's create slack/gitter workspace for pytest community?

**Comment by RonnyPfannschmidt** [comment, marked as answer]
i'm severely opposed to slack, enough that i have to suffer it at work

we already have a gitter https://gitter.im/pytest-dev/pytest -- its severely underused and sometimes doesn't see a core dev on for weeks at a time, that may get better with the recent matrix<>gitter merge

**Comment by The-Compiler** [comment]
Please let's not. It's a [walled garden](https://drewdevault.com/2015/11/01/Please-stop-using-slack.html), requires a separate instance for every slack group you're connected to (I'm in 5 or so, and it's a major pain), and it results in splitting up a rather small community even further. To recap, we already have:

- Mailinglist
- GitHub discussions
- IRC
- Gitter
- (Stackoverflow)

I really don't see a good reason to add more and more platforms just because we can, it'll just result in less and less people being at the same place.

**Reply by LaserPhaser** [reply]
Sorry - MailingLists and IRC - are quite outdated nowadays. ) 
Stackoverflow  - it's not a place for a fast discussions. 

 I did not know about Gitter , sorry.

Let's close this discussion - gitter is absolutely fine for fast discussions from my point of view.

**Reply by LaserPhaser** [reply]
maybe we should add a link to the gitter on the main page 
https://docs.pytest.org/
or in github
